modules/library/main/src/main/java/org/geotools/styling/SLDParser.java

```diff
@@ -2308,6 +2308,9 @@ private Expression parseCssParameter(Node root, boolean trimWhiteSpace) {
         for (Expression expression : expressions) {
             ret = manageMixed(ret, expression);
         }
+        if (ret == null) {
+            return ff.literal("");
```

Comment:
Double check the GTXML parser handles an empty tag in the same way or we will get into inconsistencies in our model.